unison-src/transcripts/deleteReplacements.md

```diff
@@ -23,15 +23,15 @@ x = 2
 ```
 
 ```unison
-type Foo = Foo
+unique[a] type Foo = Foo
```

Comment:
Added some of these deterministic unique types to avoid possible confusion with `()`.

Reply:
Yeah.